common/src/main/java/org/valkyrienskies/mod/mixin/mod_compat/reachentityattributes/MixinReachEntityAttributes.java

```diff
@@ -31,7 +33,8 @@ public static List<Player> getPlayersWithinReach(final Predicate<Player> viewerP
         for (final Player player : world.players()) {
             if (viewerPredicate.test(player)) {
                 final var reach = getReachDistance(player, baseReachDistance);
-                if (VSGameUtilsKt.squaredDistanceBetweenInclShips(world, x + 0.5, y + 0.5, z + 0.5, player.getX(), player.getEyeY(), player.getZ()) <= (reach * reach)) {
+                final Vec3 eye = EntityDragger.INSTANCE.serversideEyePosition(player);
```

Comment:
Why do we need a custom method for this? Why can't we use a mixin to fix the position of the player (and therefore the eye position)?

Reply:
Explained via comment.